perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -226,6 +250,28 @@ def ApiBasedBenchmarks(results, metadata, vm, storage, test_script_path,
       # User only wants to run the CLI based tests, do nothing here:
       return
 
+    def BuildCommandString(args):
```

Comment:
The way the current diff shows this function "BuildCommandString()" is embeded inside the ApiBasedBenchmark() function. Why is it so? Can you move this outside as a generic utility routine.